src/ch/blinkenlights/android/vanilla/LibraryActivity.java

```diff
@@ -139,6 +143,10 @@
 	 */
 	private long mLastActedId;
 	/**
+	 * Holds last intent that was passed to the context menu
+	 */
+	private Intent lastRequestedCtx;
+	/**
```

Comment:
This should be mLastRequestedCtx, see http://source.android.com/source/code-style.html#follow-field-naming-conventions